### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by Geartrixy**

### CODE REVIEW

File: variables_general.tf [hunk 1]
```diff
@@ -0,0 +1,34 @@
+data "aws_caller_identity" "current" {}
+
+# Account
+provider "aws" {
+  region = "${var.region}"
+}
+
+# Account ID
+variable "account_id" {
```
Comment (derBroBro) [inline variables_general.tf]: Can this be replaced by data.aws_caller_identity.current.account_id ?

File: variables_general.tf [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+}
+
+# Region
+variable "region" {
+  description = "The AWS region to run in."
+  default     = "eu-central-1"
+}
+
+# VPC ID
+variable "vpc_id" {
```
Comment (derBroBro) [inline variables_general.tf]: It looks like this is not used.

File: variables_iam_static_users.tf [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+variable "iam_user_s3_full_names" {
+  type        = "list"
+  description = "Names of the IAM users with S3 bucket full access"
+  default     = []
+}
+
+variable "iam_user_s3_list_delete" {
+  description = "Enable IAM user(s) with S3 bucket list/delete permissions"
+  default     = false
```
Comment (derBroBro) [inline variables_iam_static_users.tf]: Maybe this can be checked if the list is empty or not.

**Review by derBroBro — CHANGES_REQUESTED** [review]

File: variables_iam_static_users.tf (hunk 3 above)
Comment (Geartrixy) [reply variables_iam_static_users.tf]: todo: check if list has 0 elements instead

**Comment by Geartrixy** [comment]
@derBroBro changes made. Tested successfully in play account.

**Pull request merged**